tests/test_redcap.py

```diff
@@ -52,7 +52,7 @@ def test_read_redcap_extract_rows_and_cols(redcap_extract):
     6 rows should exist, and the output columns should match what is in our masterfile definition
     """
 
```

Comment:
Ooh and we should change Given When Then comment above to be 4 rows in redcap, 8 rows should exist at the end